src/connector/SpectodaSimulatedConnector.ts

```
@@ -0,0 +1,654 @@
+  }
+
+  connected(): Promise<SpectodaTypes.Criterium | null> {
+    logging.verbose(`connected()`);
+
+    return new Promise(async (resolve, reject) => {
+      if (this.#connected) {
+        resolve({ connector: this.type });
+      } else {
+        resolve(null);
+      }
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)